ARGO/include/TransformComponent.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include "Component.h"
+
+class TransformComponent : public Component
+{
+	struct Position
+	{
+		float x = 0.0f;
+		float y = 0.0f;
+	};
+
+	////might be good to add this to get rotation x&y
+	//struct Orientation
+	//{
+	//	float x;
+	//	float y;
+	//};
+
+	//Uncomment if needed
+	//struct Scale
+	//{
+	//	float x;
+	//	float y;
+	//};
+
```

Comment:
Is needed if commented out?

Reply:
might be needed, can be removed later once we know if we will need this or not @CathalRedmond